Makefile

```diff
@@ -1,5 +1,11 @@
+GO_PKG=github.com/openshift/insights-operator
+GO111MODULE?=on
+GOFLAGS=-mod=vendor
+export GO111MODULE
+export GOFLAGS
+
 build:
-	GO111MODULE=on go build -mod=vendor -ldflags "-X k8s.io/client-go/pkg/version.gitCommit=$$(git rev-parse HEAD) -X k8s.io/client-go/pkg/version.gitVersion=v1.0.0+$$(git rev-parse --short=7 HEAD)" -o bin/insights-operator ./cmd/insights-operator
+	GO111MODULE=on go build -ldflags "-X $(GO_PKG)/vendor/k8s.io/client-go/pkg/version.gitCommit=$$(git rev-parse HEAD) -X $(GO_PKG)/vendor/k8s.io/client-go/pkg/version.gitVersion=v1.0.0+$$(git rev-parse --short=7 HEAD)" -o bin/insights-operator ./cmd/insights-operator
```

Comment:
This is actually incorrect, had the same issue in my PR that I had to rebase (https://github.com/openshift/insights-operator/pull/244)

Reply:
And this was the actual merge conflict, introduced by https://github.com/openshift/insights-operator/pull/204